GOVERNANCE.md

```diff
@@ -360,7 +360,6 @@ If the pull request submitter is not a Core Member, the PR can be closed by any
 
 Astro features are discussed using a model called [Consensus-seeking decision-making](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making). This model attempts to achieve consensus on all significant changes to Astro, but has a fallback voting procedure in place if consensus appears unattainable.
```

Comment:
This line here mentions “a fallback voting procedure in place if consensus appears unattainable”. But below there isn’t any detail of this procedure and with the proposed change also not clear who would vote. Should we remove this line about “fallback voting”?

Reply:
I believe this is the fallback being referenced: https://github.com/withastro/.github/pull/42/files#diff-b60c6a93e9f74ee52e71bf33781c2870b8395c8c5a34f00f61305343cb8d8447R371

In the past this was a much more complicated system but we never ended up using it so IIRC we just simplified it to this new line.